.github/workflows/dart-tests.yaml

```diff
@@ -0,0 +1,90 @@
+name: Relic CI
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+      - add-dart-project-and-ci-setup # temporary, testing
+  pull_request:
+    branches:
+      - main
+      - dev
+
+env:
+  DART_VERSION: '3.3.0'
+  PUB_CACHE_PATH: ~/.pub-cache
+
+jobs:
+  dart_format:
+    name: Verify Dart Formatting
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
```

Comment:
Question: Would it be possible to add iOS to this list as well? Not sure if it is required.